optimism/src/mips/interpreter.rs

```diff
@@ -248,19 +286,19 @@ pub trait InterpreterEnv {
 
             instruction_counter + Self::constant(1)
         };
-        unsafe { self.push_register_access(idx, new_accessed.clone()) };
+        unsafe { self.push_register_access_if(idx, new_accessed.clone(), if_is_true) };
         self.add_lookup(Lookup {
             numerator: Signed {
                 sign: Sign::Pos,
-                magnitude: Self::constant(1),
+                magnitude: if_is_true.clone(),
```

Comment:
Will this "add" a lookup with magnitude zero when `if_is_true` is false?

Reply:
That's the intention, yes. It's basically a 'non-contributing lookup term'.